### ISSUE TITLE
package version detection problem

### ISSUE THREAD

**Issue by choeycui** [spyder-ide/spyder#15154]
### Issue Report Checklist

* [ ] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [ ] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [ ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ ] Tried basic troubleshooting (if a bug/error)
    * [ ] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [ ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

There is a message box occurred once the spyder is launched
![image](https://user-images.githubusercontent.com/52060363/113821383-9c75c680-97ae-11eb-9ad8-8a52b42e7e2b.png)

### What steps reproduce the problem?

1. check the condition of this message
2. 
3. 

### What is the expected output? What do you see instead?

The messagebox should not appear if the version is satisfied

### Paste Traceback/Error Below (if applicable)

```python-traceback

No Traceback is available

```

## Versions

* Spyder version: 5.0.0
* Python version: 3.7.10
* Qt version: 5.9.7
* PyQt version: 5.9.2
* Operating System name/version: win10

### Dependencies

```
The spyder-kernels 2.0.1 is already installed
```
No Traceback is available

**Comment by dovsay** [comment]
me too

Spyder 5.0.0
Python 3.8.8 
Qt version: 5.9.7
PyQt version: 5.9.2
Operating System name/version: win10 20H2

**Comment by AJJLagerweij** [comment]
I have the same issue, but it happens both for `spyder_kernels` and `rtree`.
![afbeelding](https://user-images.githubusercontent.com/31255888/113830317-2d3caa00-978f-11eb-94e0-8aecce5aa0d7.png)
A similar issue is discussed in #15153.

P.S. error/warning uses the slightly confusing name `spyder_kernels` whereas anaconda call the respective package `spyder-kernels`. Simply copying the error message into your terminal will cause conda to tell you that `spyder_kerners` does not exist.

**Comment by gb119** [comment]
Same here - something wrong with the dependency checking since it claims it needs >=2.0.1 and simultaneously tells me I have 2.0.1...
![image](https://user-images.githubusercontent.com/4428426/113841030-3aef3100-9789-11eb-9c2e-68756eaa1c8f.png)

**Comment by ccordoba12** [comment]
Hey all, thanks for reporting. This is a duplicate of issue #15091 and it'll be fixed in our next version (5.0.1), to be released shortly.

In the meantime, please ignore that message, it's a false alarm.
